src/test/java/com/solvd/shop24/BaseTest.java

```diff
@@ -0,0 +1,17 @@
+package com.solvd.shop24;
+
+import com.qaprosoft.carina.core.foundation.AbstractTest;
+import com.solvd.shop24.gui.pages.shop24.HomePage;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+
+public class BaseTest extends AbstractTest {
+
+    protected HomePage openHomePage(WebDriver driver) {
```

Comment:
you need to move logic to basetest methods only if that really needed.
not like here. you don't complete any specific steps in that method.
redundant basetest methods make test less transparent. try to avoid them